build broken on ppc64/ppc64le after recent change

seems commit a35ff787bfa75a655dedd7a0ca8cf1fcd1d58619 broke build on ppc64/ppc64le
````
...
  CC       dwarf/global.lo
  CCLD     libunwind-dwarf-common.la
  CCLD     libunwind-dwarf-local.la
  CC       elf64.lo
In file included from elf64.c:3:
elfxx.c: In function '_Uelf64_lookup_symbol_closeness':
elfxx.c:165:43: error: 'as' undeclared (first use in this function); did you mean 'abs'?
  165 |                   if (tdep_get_func_addr (as, val, &val) < 0)
      |                                           ^~
      |                                           abs
elfxx.c:165:43: note: each undeclared identifier is reported only once for each function it appears in
make[2]: *** [Makefile:4756: elf64.lo] Error 1
make[2]: Leaving directory '/home/jenkins/workspace/libunwind/label/ppc64le/src'
make[1]: *** [Makefile:2889: all] Error 2
make[1]: Leaving directory '/home/jenkins/workspace/libunwind/label/ppc64le/src'
make: *** [Makefile:619: all-recursive] Error 1
````

--- a/src/elfxx.c
+++ b/src/elfxx.c
@@ -113,9 +113,10 @@ elf_w (string_table) (const struct elf_image *ei, int section)
 }
 
 static int
-elf_w (lookup_symbol_closeness) (const struct symbol_lookup_context *context,
-                                int (*callback)(const struct symbol_lookup_context *context,
-                                                const struct symbol_info *syminfo, void *data),
+elf_w (lookup_symbol_closeness) (unw_addr_space_t as UNUSED,
+                                 const struct symbol_lookup_context *context,
+                                 int (*callback)(const struct symbol_lookup_context *context,
+                                                 const struct symbol_info *syminfo, void *data),
                                  void *data)
 {
   struct elf_image *ei = context->ei;
@@ -213,7 +214,7 @@ elf_w (lookup_symbol_callback)(const struct symbol_lookup_context *context,
 }
 
 static int
-elf_w (lookup_symbol) (unw_addr_space_t as UNUSED,
+elf_w (lookup_symbol) (unw_addr_space_t as,
                        unw_word_t ip, struct elf_image *ei,
                        Elf_W (Addr) load_offset,
                        char *buf, size_t buf_len, Elf_W (Addr) *min_dist)
@@ -231,7 +232,8 @@ elf_w (lookup_symbol) (unw_addr_space_t as UNUSED,
       .buf = buf, 
       .buf_len = buf_len,
     };
-  return elf_w (lookup_symbol_closeness) (&context,
+  return elf_w (lookup_symbol_closeness) (as,
+                                          &context,
                                           elf_w (lookup_symbol_callback),
                                           &data);
 }
@@ -256,7 +258,7 @@ elf_w (lookup_ip_range_callback)(const struct symbol_lookup_context *context,
 }
 
 static int
-elf_w (lookup_ip_range)(unw_addr_space_t as UNUSED,
+elf_w (lookup_ip_range)(unw_addr_space_t as,
                         unw_word_t ip, struct elf_image *ei,
                         Elf_W (Addr) load_offset, Elf_W (Addr) *start_ip,
                         Elf_W (Addr) *end_ip, Elf_W (Addr) *min_dist)
@@ -274,7 +276,8 @@ elf_w (lookup_ip_range)(unw_addr_space_t as UNUSED,
       .start_ip = start_ip,
       .end_ip = end_ip
     };
-  return elf_w (lookup_symbol_closeness) (&context,
+  return elf_w (lookup_symbol_closeness) (as,
+                                          &context,
                                           elf_w (lookup_ip_range_callback),
                                           &data);
 }

